README.md

```diff
@@ -8,25 +8,26 @@
 
 An open-source, feature-packed alternative bootstrapper for Roblox.
 
-This a drop-in replacement for the stock Roblox bootstrapper, working more or less how you'd expect it to, while providing additional useful features. This does not touch or modify the game client itself, it's just a launcher! So don't worry, there's practically no risk of being banned for using this.
+>[!NOTE]
```

Comment:
I don't really like the use of a banner section here, I might remove that.

Reply:
understood, if you want, i can change that too